app/api/api_v1/routers/search.py

```diff
@@ -68,9 +69,14 @@ def _search_request(
         if use_vespa:
             data_access_search_params = create_vespa_search_params(db, search_body)
             # TODO: we may wish to cache responses to improve pagination performance
-            data_access_search_response = _VESPA_CONNECTION.search(
-                parameters=data_access_search_params
-            )
+            try:
+                data_access_search_response = _VESPA_CONNECTION.search(
+                    parameters=data_access_search_params
+                )
+            except QueryError:
```

Comment:
Do we not want except as e and then pass the error into the HttpException response?

Reply:
Valid question. We spoke about this and thought it might be a good future improvement